R/setup.R

```diff
@@ -1,44 +1,117 @@
 #' Setup deployment for Travis CI
 #'
 #' Creates a public-private key pair,
-#' adds the public key to the GitHub repository via [github_add_key()],
+#' adds the public key to the GitHub repository via `github_add_key()`,
```

Comment:
Is this still true?

Reply:
Yes, we just do not export `github_add_key()` anymore.

Reply:
But is it relevant for the user documentation?